residual_block_architecture.py

```
@@ -0,0 +1,260 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
```

Ruff (0.12.2)

`torch.nn.functional` imported but unused

Remove unused import: `torch.nn.functional`

(F401)